### PR TITLE
Use `storm.local.dir` for generated-conf path.

### PR DESCRIPTION
**Pull request by brndnmtthws**
to address #70.

### CODE REVIEW

File: src/main/storm/mesos/MesosNimbus.java [hunk 1]
```diff
@@ -152,16 +153,21 @@ protected void initialize(Map conf, String localDir) throws Exception {
     }
     _container = Optional.fromNullable((String) conf.get(CONF_MESOS_CONTAINER_DOCKER_IMAGE));
     _scheduler = new NimbusScheduler(this);
+
+    // Generate YAML to be served up to clients
+    _generatedConfPath = Paths.get(
+        Optional.fromNullable((String) conf.get(Config.STORM_LOCAL_DIR)).or("./"),
+        "generated-conf");
+    if (!_generatedConfPath.toFile().exists() && !_generatedConfPath.toFile().mkdirs()) {
+      throw new RuntimeException("Couldn't create generated-conf dir");
```
Comment (erikdw) [inline src/main/storm/mesos/MesosNimbus.java]: nice, you fixed that greate typo ;)

Comment (erikdw) [inline src/main/storm/mesos/MesosNimbus.java]: Maybe add in the path that couldn't be created to the exception?  ` _generatedConfPath`

Comment (brndnmtthws) [inline src/main/storm/mesos/MesosNimbus.java]: Done.

Comment (erikdw) [inline src/main/storm/mesos/MesosNimbus.java]: Thanks for always being so quick!

**Comment by erikdw** [comment]
@brndnmtthws : looks good, I think this is an improvement, but perhaps we should wait for @salimane to test it to ensure it actually fixes his issue in #70.

**Comment by salimane** [comment]
@brndnmtthws @erikdw everything looks good, tested it and it works :+1: 
Thanks

**Comment by erikdw** [comment]
thanks @brndnmtthws & @salimane !   :ship: :it: !

**Pull request merged**